app/helpers/check_answers_helper.rb

```diff
@@ -1,102 +1,52 @@
 module CheckAnswersHelper
-  SKIPPABLE_QUESTIONS = %w[
-    are_you_a_manufacturer
-    product_details
-    rooms_number
-    accommodation_cost
-    transport_type
-    transport_cost
-    offer_space_type
-    space_cost
-    offer_care_qualifications
-    care_cost
-    offer_staff_type
-    offer_staff_charge
-    construction_services
-    construction_services_other
-    construction_cost
-    it_services
-    it_services_other
-    it_cost
-  ].freeze
-
-  def items
-    items = questions.map do |question|
-      # We have answers as strings and hashes. The hashes need a little more
-      # work to make them readable.
-
-      next if skip_question?(question)
-
-      next product_details if question.eql?("product_details")
-      next transport_type if question.eql?("transport_type")
-      next offer_care_qualifications if question.eql?("offer_care_qualifications")
-      next offer_staff_type if question.eql?("offer_staff_type")
-      next link_to_parent_page(question, "offer_staff_type") if question.eql?("offer_staff_charge")
-      next link_to_parent_page(question, "construction_services") if question.eql?("construction_services_other")
-      next link_to_parent_page(question, "construction_services") if question.eql?("construction_cost")
-      next link_to_parent_page(question, "it_services") if question.eql?("it_services_other")
-      next link_to_parent_page(question, "it_services") if question.eql?("it_cost")
-      next link_to_parent_page(question, "transport_type") if question.eql?("transport_cost")
-      next link_to_parent_page(question, "offer_care_qualifications") if question.eql?("care_cost")
-      next link_to_parent_page(question, "rooms_number") if question.eql?("accommodation_cost")
-      next link_to_parent_page(question, "offer_space_type") if question.eql?("space_cost")
-
-      value = concat_answer(session[question], question)
-      [{
-        field: t("coronavirus_form.questions.#{question}.title"),
-        value: value.is_a?(Array) ? value.map { |v| sanitize(v) } : sanitize(value),
-        edit: {
-          href: "#{question.dasherize}?change-answer",
-        },
-        delete: {
-          href: "#{question.dasherize}?change-answer",
-        },
-      }]
+  def summary_entry(session_path_or_key, description, edit, delete)
```

Comment:
It looks like `edit` and `delete` are booleans. I think they should be named parameters so when when the method is called it's clear what the values represent. I'd also give them default values so they only need to be passed in exceptional circumstances. 

i.e.
```
def summary_entry(session_path_or_key, description, edit: true, delete: false)
```